### DISCUSSION TITLE
Hey how do you get started as a programmer?

### DISCUSSION THREAD

**Discussion by Trash4468** [community/community#66847] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What are some tips for beginners to improve their coding skills?

**Comment by fruitychan** [comment]
get good

**Comment by Trash4468** [comment]
.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
